### ISSUE TITLE
MIO168+AFE3: AIN3+AIN4 Power calculation is off?

### ISSUE THREAD

**Issue by fietser28** [eez-open/modular-psu-firmware#305]
![image](https://user-images.githubusercontent.com/8796121/172062473-0f46eee6-372d-400c-a71f-b7d88d2368e4.png)
P2 should read 0,4889 W.
DC power source is used, all numbers are stable.

AC analysis is correct:
![image](https://user-images.githubusercontent.com/8796121/172062543-a5380b60-2ec3-4324-be67-88ed94456925.png)

**Comment by fietser28** [comment]
Power function is using uncalibrated values.

**Comment by MartinNPN** [comment]
Couldn't this closed after https://github.com/eez-open/modular-psu-firmware/pull/306 is merged ?

**Comment by fietser28** [comment]
> Couldn't this closed after [#306](https://github.com/eez-open/modular-psu-firmware/pull/306) is merged ?

Yes, it is in the master branch. But I don't think it is in the current release (1.7.x). Not sure though.
